contracts/rbn-staking/Redeemer.sol

```diff
@@ -75,19 +75,20 @@ contract Redeemer is Ownable {
   }
 
   /**
-   * @dev Redeems the rbn
-   * @param _amount is the amount
+   * @dev Redeems the rbn using seizer implementation logic
    */
-  function redeemRBN(uint256 _amount) external onlyOwner {
+  function redeemRBN() external onlyOwner {
     require(
       address(votingEscrowContract) != address(0),
       "votingEscrowContract is 0x0"
     );
+    require(
+      address(seizerImplementation) != address(0),
+      "seizerImplementation is 0x0"
+    );
```

Comment:
We don't have to check whether the `seizerImplementation` is not address(0) as long as we check in the functions which set `seizerImplementation`

Reply:
at first seizerImplementation will be address(0). but I changed this anyway bc I doubt we will call that redeemRBN method knowing siezer implementation is address(0).